src/pages/TokenDetails/index.tsx

```diff
@@ -53,13 +54,7 @@ const RightPanel = styled.div`
     display: none;
   }
 `
-const Widget = styled.div`
-  height: 348px;
-  width: 284px;
```

Comment:
widget has min width of 300 but has go-ahead from design to change minwidth to 275. hasn't been done yet though and is an open task on Jira 
https://uniswaplabs.atlassian.net/browse/DE-99?atlOrigin=eyJpIjoiNzBiNDE0ZDJiYTkwNGMwOWFjNWZlMGU4N2EyZWNiMWIiLCJwIjoiaiJ9